scripts/waifulib/glslc.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
Recommended way is use `#!/usr/bin/env python3` if using python 3 or `#!/usr/bin/env python2` if using python 2. Do not use `#!/usr/bin/env python`.

Reply:
That doesn't make sense. Waf supports every Python version since 2.5.
Moreover, this file doesn't meant to be executed, so this line could be safely removed.

Reply:
Then remove.